### ISSUE TITLE
mobille-accessibility not working with iFrame content with Android

### ISSUE THREAD

**Issue by ravidixit** [phonegap/phonegap-mobile-accessibility#8]
I am working on an Android phone gap application which loads few content in iFrame from a web server, 

On using the phonegap-mobile-accessibility I found that it fails to read out the content from the iFrame which fails to make the app as 508 compliance.

Could you please guide me out to read out the selected content from the iFrame using this plugin in Android.
